Weather Alert/Networking/Networking.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Networking.swift
+//  Weather Alert
+//
+//  Created by Jack Hopkins on 18/11/2016.
+//  Copyright © 2016 UKFast. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+open class NetworkingRequest {
+    
+    // MARK: Variables
+    
+    public var path: String
+    public var response: [String : Any]?
+    
+    var networking: Networking { return AppShared.instances.networking }
+    var dataController: DataController { return AppShared.instances.dataController }
+    
+    init() {
+        path = "weather?id=1000006"
```

Comment:
Is this `id` should be hardcoded here? Not sure what it does, but it seems wrong :)
Also, you've created a NetworkRequest (generic) class, but your path is specifically for weather?  Maybe this path should be empty for the base class